code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -240,35 +250,52 @@
 	icon_state = "sec"
 
 /obj/structure/closet/secure_closet/security/fill()
-	//Appearance
+	// Appearance
 	if(prob(50))
 		new /obj/item/storage/backpack/security(src)
 	else
 		new /obj/item/storage/backpack/satchel/sec(src)
+
+	// Backpacks
 	new /obj/item/storage/backpack/duffel/sec(src)
-	new /obj/item/clothing/suit/armor/carrier/officer(src)
+
+	// Carrier Accessories
+	spawn_armor()
 	new /obj/item/clothing/accessory/arm_guard(src)
 	new /obj/item/clothing/accessory/leg_guard(src)
-	new /obj/item/clothing/head/helmet/security(src)
 	new /obj/item/clothing/accessory/badge/officer(src)
+	new /obj/item/clothing/accessory/holster/modular(src)
+	new /obj/item/clothing/accessory/storage/modular_pouch(src)
+
+	new /obj/item/clothing/head/helmet/security(src)
+
 	new /obj/item/clothing/mask/gas/alt(src)
 	new /obj/item/clothing/mask/gas/half(src)
-	//Tools
+
+	// Tools
 	new /obj/item/device/radio/headset/headset_sec(src)
 	new /obj/item/device/radio/headset/headset_sec/alt(src)
 	new /obj/item/clothing/glasses/sunglasses/sechud/aviator(src)
 	new /obj/item/clothing/glasses/sunglasses/sechud(src)
+
 	new /obj/item/gun/energy/disruptorpistol/security(src)
 	new /obj/item/taperoll/police(src)
 	new /obj/item/device/hailer(src)
 	new /obj/item/device/flashlight/flare/glowstick/red(src)
-	//Belts
+
 	new /obj/item/clothing/accessory/storage/black_vest(src)
 	new /obj/item/clothing/accessory/holster/hip(src)
-	new /obj/item/clothing/accessory/holster/modular(src)
 	new /obj/item/storage/belt/security/full(src)
 	new /obj/item/clothing/suit/storage/hazardvest/security/officer(src)
 
+// this is the locker that spawns in the locker room
+/obj/structure/closet/secure_closet/security/proc/spawn_armor()
+	new /obj/item/clothing/accessory/armor_plate(src)
+
+// this is the locker that spawns in the security checkpoints around the ship
+/obj/structure/closet/secure_closet/security/spare/spawn_armor()
+	new /obj/item/clothing/suit/armor/carrier/officer(src)
```

Comment:
really, *really* not a fan of this approach with this proc and `/spare` locker

Reply:
also if sec officer lockers start with only carriers and no vests, you can't quickly give armor to non-sec or even just cadets (and have to run to a checkpoint instead)

imo sec lockers should still have vest (as in both vests and plates, just separate)